python/lsst/ts/adamSensors/model.py

```diff
@@ -21,35 +24,38 @@ class AdamModel:
             the pymodbus object representing the ADAM 6024
     """
 
-    def __init__(self, log=None, simulation_mode=False):
+    def __init__(self, ip, port, log=None, simulation_mode=False):
+        self.loop = asyncio.new_event_loop()
+        self.clientip = ip
```

Comment:
Does this model have to keep track of any other port and IP address? If not, then please consider ditching the "client" in "clientip" and "clientport" since it adds no information.